odoo/addons/base/views/ir_qweb_widget_templates.xml

```diff
@@ -25,7 +25,7 @@
         <div class="gap-2" itemprop="address" itemscope="itemscope" itemtype="http://schema.org/PostalAddress">
             <div t-if="address and 'address' in fields" t-attf-class="d-flex align-items-baseline gap-1">
                 <i t-if="not options.get('no_marker')" class="fa fa-map-marker fa-fw" role="img" aria-label="Address" title="Address"/>
-                <span class="d-block w-100 lh-sm" itemprop="streetAddress" t-esc="address"/>
+                <span class="d-block lh-sm text-truncate" itemprop="streetAddress" t-esc="address"/>
```

Comment:
@aath-odoo Forgot to add one more point. It's extra not related to our spec. Can we remove `d-block` from here as it is already applied from the SCSS file? Can you please check?